packages/cf-component-dynamic-content/src/DynamicContent.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ theme }) => ({
+  display: theme.display
```

Comment:
It looks like we override `color` and `textDecoration` internally. Will we still be able to do so here?